OpenIPSL/UsersGuide/package.mo

```diff
@@ -9,6 +9,7 @@ The <b>Open-Instance Power System Library - OpenIPSL</b> contains many power sys
 </p>
 <p>
 This is a very short <b>User's Guide</b> that will try to help users to get familiar with the library providing general information about the <b>OpenIPSL</b>.
+More information about the library can be found in this <a href=\"https://openipsl.readthedocs.io/en/latest/\">website</a> dedicated to the OpenIPSL organization. 
 </p>
```

Comment:
I think you got the wrong address here. readthedocs should not appear anywhere. The content should actually be removed. It is openipsl.org that is the correct address.

Reply:
is this what you meant?